lib/matplotlib/backends/backend_svg.py

```diff
@@ -912,86 +922,46 @@ def _draw_text_as_path(self, gc, x, y, s, prop, angle, ismath, mtext=None):
         style = {}
         if color != '#000000':
             style['fill'] = color
-
         alpha = gc.get_alpha() if gc.get_forced_alpha() else gc.get_rgb()[3]
         if alpha != 1:
             style['opacity'] = short_float_fmt(alpha)
+        font_scale = fontsize / text2path.FONT_SCALE
+        attrib = {
+            'style': generate_css(style),
+            'transform': generate_transform([
+                ('translate', (x, y)),
+                ('rotate', (-angle,)),
+                ('scale', (font_scale, -font_scale))]),
+        }
+        writer.start('g', attrib=attrib)
```

Comment:
Is this the same element as previously l.944? If so

- why is it pulled out out of the `if not ismath`?
- AFAICS the `devs` element created in `_update_glyph_map_defs` is now a child of `g`, which it had not been before.

Reply:
a) to just have a single (begin-group, end-group) in the program flow.
b) per https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs https://www.w3.org/TR/SVG2/struct.html#Head https://www.w3.org/TR/SVGTiny12/struct.html#DefsElement we can put \<def> anywhere -- it's suggested to collect them all to the top of the file, but we don't even do that (that can't really work with the current design as we're streaming out the svg "on the fly").